charts/prometheus-node-exporter/Chart.yaml

```diff
@@ -6,8 +6,8 @@ keywords:
   - prometheus
   - exporter
 type: application
-version: 4.6.0
-appVersion: 1.3.1
+version: 4.6.1
```

Comment:
imho chart minor version should be raised, if app minor version has changed

Reply:
So you think we should bump it to 4.7.0 ?

Reply:
yes

Reply:
done